fdbrpc/md5/md5.h

```diff
@@ -41,9 +41,9 @@ typedef struct {
 #ifdef __cplusplus
 extern "C" {
 #endif
-	void MD5_Init(MD5_CTX *ctx);
-	void MD5_Update(MD5_CTX *ctx, const void *data, unsigned long size);
-	void MD5_Final(unsigned char *result, MD5_CTX *ctx);
+	void __attribute__((weak)) MD5_Init(MD5_CTX *ctx);
```

Comment:
I think you'll need to add a preprocessor macro, probably to flow/Platform.h, which looks like 

```
#ifndef MULTIPLY_DEFINED_SYMBOL
#if windows
#define MULTIPLY_DEFINED_SYMBOL __declspec(selectany)
#else
#define MULTIPLY_DEFINED_SYMBOL __attribute__((weak))
#endif 
#endif 
```

and then use MULTIPLY_DEFINED_SYMBOL, or some better name, here, as msvc doesn't support __attribute__((weak)).